platform/linux-generic/Makefile.am

```diff
@@ -370,7 +372,9 @@ odpapiabiarchinclude_HEADERS += arch/default/odp/api/abi/atomic_generic.h \
 				arch/aarch64/odp/api/abi/cpu.h \
 				arch/aarch64/odp/api/abi/sync_inlines.h \
 				arch/common/odp/api/abi/time_cpu_inlines.h \
-				arch/aarch64/odp/api/abi/time_inlines.h
+				arch/aarch64/odp/api/abi/time_inlines.h \
+				arch/default/odp/api/abi/ticketlock_generic.h \
```

Comment:
Since using WFE may have performance impacts, adding a configuration option for enabling/disabling WFE usage could be useful. So for example like [this](https://github.com/MatiasElo/odp/commit/61ccf2565adc8e5b1016031d89b8d8e15c0eb066)  and then one can pick the included header here based on `WFE_LOCKS` value.